router/router_system.go

```diff
@@ -139,5 +151,7 @@ func postUpdateConfiguration(c *gin.Context) {
 	// Since we wrote it to the disk successfully now update the global configuration
 	// state to use this new configuration struct.
 	config.Set(cfg)
-	c.Status(http.StatusNoContent)
+	c.JSON(http.StatusOK, postUpdateConfigurationResponse{
```

Comment:
The panel doesn't seem to care about the specific response code, so this should be non-breaking.